docs/ossm/kiali-multi-cluster/README.md

```diff
@@ -0,0 +1,145 @@
+[Return to OSSM Docs](../)
+
+# Kiali - multi-cluster
+
+For multi-cluster Istio deployments, Kiali can show you a single unified view of your mesh across clusters.
+
+Before proceeding with the setup, ensure you meet the requirements.
+
+## Requirements
+
+- Two OpenShift clusters. In this tutorial they are named `east` and `west`.
+- Istio installed in a multi-cluster configuration on each cluster.
+- IstioCNI installed on each cluster.
+- Aggregated metrics and traces. Kiali needs a single endpoint for metrics and a single endpoint for traces where it can consume aggregated metrics/traces across all clusters. There are multiple ways to aggregate metrics/traces such as Prometheus federation or using OTEL collector pipelines.
+- Cluster Admin for each cluster.
+- Kiali Operator v1.89 installed on the `east` cluster.
+
+## Setup
+
+In this tutorial, we will deploy Kiali on the `east` cluster and then grant Kiali access to the `west` cluster. The unified multi-cluster setup requires a Kiali Service Account (SA) to have read access to each Kubernetes cluster in the mesh. This is separate from the user credentials that are required when a user logs into Kiali. Kiali uses the user's credentials to check if the user has access to a namespace and when performing any write operation such as creating/editing/deleting objects in Kubernetes. To give the Kiali Service Account access to each remote cluster, a kubeconfig with credentials needs to be created and mounted into the Kiali pod.
+
+### Procedure
+
+1. Install Kiali on the `east` cluster.
+
+   Create a file named `kiali.yaml`.
+
+   ```yaml
+   apiVersion: kiali.io/v1alpha1
+   kind: Kiali
+   metadata:
+     name: kiali
+     namespace: istio-system
+   spec:
+     version: v1.89
+   ```
+
+   Apply the yaml file into the `east` cluster.
+
+   ```sh
+   oc --context east apply -f kiali.yaml
+   ```
+
+   Wait for Kiali to become ready.
+
+   ```sh
+   oc --context east wait --for=condition=Successful --timeout=2m kialis/kiali -n istio-system
+   ```
+
+1. Create an `OAuthClient` on the remote cluster so that Kiali can access the OpenShift API server on behalf of users.
+
+   Find your Kiali route's hostname.
+
+   ```sh
+   oc --context east get route kiali -n istio-system -o jsonpath='{.spec.host}'
+   ```
+
+   Create a file named `oauthclientwest.yaml`
+
+   ```yaml
+   apiVersion: oauth.openshift.io/v1
+   grantMethod: auto
+   kind: OAuthClient
+   metadata:
+     labels:
+       app: kiali
+       app.kubernetes.io/instance: kiali
+       app.kubernetes.io/name: kiali
+       app.kubernetes.io/part-of: kiali
+     name: kiali-istio-system
+   redirectURIs:
+     - https://<your-kiali-route-hostname>/api/auth/callback/west
```

Comment:
This is the `OAuthClient` that gets created in the `west` cluster that redirects your west cluster login back to Kiali in the `east` cluster. It needs to have `west` appended on the end of the url since that tells the Kiali server which cluster the redirect is for.